src/modules/cpu/kernel/color_jitter.hpp

```diff
@@ -0,0 +1,1119 @@
+#include "rppdefs.h"
+#include "cpu/rpp_cpu_simd.hpp"
+#include "cpu/rpp_cpu_common.hpp"
+
+RppStatus color_jitter_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                         RpptDescPtr srcDescPtr,
+                                         Rpp8u *dstPtr,
+                                         RpptDescPtr dstDescPtr,
+                                         Rpp32f *brightnessTensor,
+                                         Rpp32f *contrastTensor,
+                                         Rpp32f *hueTensor,
+                                         Rpp32f *saturationTensor,
+                                         RpptROIPtr roiTensorPtrSrc,
+                                         RpptRoiType roiType,
+                                         RppLayoutParams layoutParams)
+{
+    RpptROI roiDefault = {0, 0, (Rpp32s)srcDescPtr->w, (Rpp32s)srcDescPtr->h};
+    RpptROIPtr roiPtrDefault = &roiDefault;
+
+    omp_set_dynamic(0);
+#pragma omp parallel for num_threads(dstDescPtr->n)
+    for(int batchCount = 0; batchCount < dstDescPtr->n; batchCount++)
+    {
+        RpptROI roi;
+        RpptROIPtr roiPtr = &roi;
+        RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
+        compute_roi_validation_host(roiPtrInput, roiPtr, roiPtrDefault, roiType);
```

Comment:
you can pass the address of the roi's directly to the function instead of storing them in separate variables

Reply:
Done